src/pages/documentation/release-notes/index.md

```diff
@@ -3,6 +3,21 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 ---
 
 # Release notes
+## April 6, 2023
+
+### Flutter EdgeBridge plugin 1.0.0
+
+The Adobe Experience Platform Edge Bridge Flutter plugin is now available.
+With this release, the extension enables a drop-in solution for converting MobileCore track API (trackAction & trackState) generated events into Edge Network events.
```

Comment:
"... for converting the Mobile Core track API generated events (trackAction and trackState) into Edge Network events."

I know that we're trying to say that they're related to the track APIs, hence why we had the bracket there before, but I think it makes it less clear, if phrased that way, that it's converting the generated events from the Mobile Core track APIs, rather than converting the APIs themselves.

Reply:
Also, any reason that this is on a newline? If we want this as two paragraphs, I think it needs to have another newline above it (so the double newline).

Reply:
We want to say Edge Bridge converts the events which generated by tractAction and trackState into Edge Network events.  Does it make sense?

Reply:
Actually, let's try this instead - "... for converting generated events from the Mobile Core track APIs (trackAction and trackState) into Edge Network events." That way we don't have the awkward brackets between the full phrase, but still have it phrased in a nice way.

Reply:
Updated!